streaming_providers/realdebrid/utils.py

```
@@ -18,15 +20,17 @@ def create_download_link(rd_client, torrent_info, filename, file_index, episode)
         True,
     )
     try:
-        response = rd_client.create_download_link(torrent_info["links"][file_index])
+        response = await rd_client.create_download_link(
+            torrent_info["links"][file_index]
+        )
     except IndexError:
         raise ProviderException(
             "No matching file available for this torrent", "no_matching_file.mp4"
         )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)